depends/README.md

```diff
@@ -28,6 +28,18 @@ Common `host-platform-triplets` for cross compilation are:
 
 No other options are needed, the paths are automatically configured.
 
+Install the required dependencies: Ubuntu & Debian
+--------------------------------------------------
+
+For macOS/Win32/Win64 cross compilation:
+
+    sudo apt-get install curl librsvg2-bin libtiff-tools bsdmainutils cmake imagemagick libcap-dev libz-dev libbz2-dev mingw-w64 g++-mingw-w64 nsis zip python-setuptools
```

Comment:
I think this is duplicated from the windows build instructions, might be better to refer to there instead?

Reply:
Oh. Right.
Fixed.